src/puptoo/process.py

```diff
@@ -193,6 +193,11 @@ def system_profile(
     if redhat_release:
         try:
             profile["os_release"] = redhat_release.rhel
+            profile["operating_system"] = {
+                "major": redhat_release.major,
+                "minor": redhat_release.minor,
+                "name": "RHEL"
```

Comment:
There is a "redhat_release.is_rhel" in the parser that you can use here:

```python
    @property
    def is_rhel(self):
        """bool: True if this OS belong to RHEL, else False."""
        return "Red Hat Enterprise Linux" in self.parsed["product"]
```

Reply:
Leaving as-is as per slack convo since RHEL is the only option here at this time.  Can revisit as more options are included.